ast/ast.go

```diff
@@ -37,6 +37,8 @@ type Node interface {
 	Text() string
 	// SetText sets original text to the Node.
 	SetText(text string)
+	// SecureText is different from Text that it hide password information.
+	SecureText() string
```

Comment:
I think this method should not be put in Node interface.
How about defining a CrusialStmt interface which only has a SecureText method?

Reply:
So we implement `SecureText()` for so many

```
*ast.CreateUserStmt, *ast.DropUserStmt, *ast.AlterUserStmt, *ast.SetPwdStmt, *ast.GrantStmt,
*ast.RevokeStmt, *ast.AlterTableStmt, *ast.CreateDatabaseStmt, *ast.CreateIndexStmt, *ast.CreateTableStmt,
*ast.DropDatabaseStmt, *ast.DropIndexStmt, *ast.DropTableStmt, *ast.RenameTableStmt, *ast.TruncateTableStmt:
```

and most of those implemention is just ... ?

```
func (x *XXX) SecureText() string {
    return x.Text()
}
```

Reply:
Well, not all crucial statement need to have secure text.
Then how about define a `SensitiveStatement` which has `SecureText`?

Reply:
```
if ss, ok := node.(SensitiveStmt) {
    text = ss.SecureText()
} else {
    text = node.Text()
}
```

Reply:
Could we add SecureText to StmtNode interface?

Reply:
@shenli 
`SecureText` only needed by 4 statements, I think adding it to all StmtNode is not worthy.

Reply:
ok